pipelines/broad/rna_seq/BroadInternalRNAWithUMIsPipeline.wdl

```diff
@@ -0,0 +1,107 @@
+version 1.0
+
+import "RNAWithUMIsPipeline.wdl" as RNAWithUMIsPipeline
```

Comment:
I *think* you may need to make this "../../../pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl" to make it work with one of the build scripts (could be wrong).